### PR TITLE
Add controllers

### PR DESCRIPTION
**Pull request by suneettipirneni**
This adds and transfers relative code to the new controller objects.

### CODE REVIEW

File: src/controllers/group.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import GroupModel from '../models/group';
+
+const Group = {
+  register: async (req, res) => {
+    const newGroup = new GroupModel(
+      {
+        inviteCode: req.body.inviteCode,
```
Comment (Armintron) [inline src/controllers/group.js:7]: Invitecode is not given by the request.

**Review by Armintron — CHANGES_REQUESTED** [review]

File: src/controllers/group.js (hunk 1 above)
Comment (suneettipirneni) [reply src/controllers/group.js:7]: This was just code transferred from whatever was in main, I didn't add this.

**Comment by Armintron** [comment]
Do you want to wait until my PR is approved and then redo this PR with the updated code?

**Comment by suneettipirneni** [comment]
> Do you want to wait until my PR is approved and then redo this PR with the updated code?

Sure

**Review by Armintron — APPROVED** [review]
Looks good to me

**Pull request merged**